src/assets/data/SearchCarouselData.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "name": "Get that perfect fit",
```

Comment:
Here we are supporting only english. Check how we can support multiple languages for static json and ping me the solution you came up with

Reply:
I have fixed it as discussed in the call.